feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -14,7 +19,6 @@ import {
 } from '@spartacus/storefront';
 import { BehaviorSubject, Observable } from 'rxjs';
 import { finalize, pluck } from 'rxjs/operators';
-
 @Component({
   selector: 'cx-import-entries-dialog',
   templateUrl: './import-entries-dialog.component.html',
```

Comment:
> Opened modals should allow to close them on ESC key press.

1 I can not close import dialog by escape after upload (when seeing summary) - need to click the mouse on modal first or press tab first (but it will work only if `show more` button exists)

I think trigger focus to the element is necessary when finished.

2 I can still use escape while uploading when click on modal first (or press tab if `show more` button exists)

something like this could help for point 2:
![image](https://user-images.githubusercontent.com/8040837/159805966-9194a80d-9398-48f6-8e88-4595be1437bb.png)

Reply:
I fixed point 1, and used your suggestions for point to prevent the dialog from closing while uploading